mod_ngarn/worker.py

```diff
@@ -77,18 +79,23 @@ class Job:
 
 @dataclass
 class JobRunner:
-    async def fetch_job(self, cnx: asyncpg.Connection):
+    async def fetch_job(
+        self,
+        cnx: asyncpg.Connection,
+        table: str = escape_table_name(os.getenv('DBTABLE', 'modngarn_job')),
+    ):
+
         result = await cnx.fetchrow(
-            """
-            SELECT id, fn_name, args, kwargs, priority
-            FROM modngarn_job 
+            """SELECT id, fn_name, args, kwargs, priority FROM {table}
```

Comment:
We cannot use $1 here right?

Reply:
yes

Reply:
Probably you want to put the table name in double quotes though.